recipes/phylophlan/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "phylophlan" %}
+{% set version = "3.0" %}
+{% set sha256 = "1a6f19748f5d7ae14b1a2b594598b8dd7ba02c3373ad8ebb0bdaa2e93dbe98a7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/biobakery/phylophlan/archive/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+  # path: ..
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+    - biopython >=1.73
```

Comment:
Starting from `biopython`, these dependencies shouldn’t be necessary at build time, so you should be able to remove them from the `host:` section.